pf4j/src/main/java/org/pf4j/AbstractExtensionFinder.java

```diff
@@ -176,6 +158,20 @@ public AbstractExtensionFinder(PluginManager pluginManager) {
         return result;
     }
 
+    private boolean pluginNotStarted(String pluginId) {
+        if (pluginId != null) {
+            PluginWrapper pluginWrapper = pluginManager.getPlugin(pluginId);
+            if (PluginState.STARTED != pluginWrapper.getPluginState()) {
+                return true;
+            }
+
+            log.trace("Checking extensions from plugin '{}'", pluginId);
```

Comment:
I think that the log messages related to extensions cannot be part of "pluginNotStarted"